mesonbuild/backend/xcodebackend.py

```diff
@@ -18,7 +18,6 @@
 
 from . import backends
 from .. import build
-from .. import dependencies
```

Comment:
This needs to be folded into the commit that dropped using the import, or it breaks git bisect.

Note the commit message for that one is inaccurate: it says "This is safer and matches what the vs backend does." but the actual bug it fixes is that if dependencies.platform.AppleFrameworks has never been imported yet, it crashes.

Reply:
I have amended the latter commit accordingly and dropped the former one.

Reply:
Thanks, pulled this commit as 665275a78506b68638f4e996ef8e8b8e549d5c8f.